src/python/pants/source/source_root.py

```diff
@@ -172,6 +179,19 @@ def register_options(cls, register):
             "See https://pants.readme.io/docs/source-roots.",
         )
 
+        register(
+            "--marker-filename",
```

Comment:
Should this be a list rather than a single file? From a glob expansion perspective it's the same cost, because we operate at the level of directory listings.

Reply:
Yeah, I suppose a repo might want to use existing setup.py files for the python source roots, and some other name for JVM source roots or whatever. Will make this a list.